src/gui/guiFormSpecMenu.cpp

```diff
@@ -1564,11 +1782,34 @@ void GUIFormSpecMenu::parseBackgroundColor(parserData* data, const std::string &
 
 	if (((parts.size() == 1) || (parts.size() == 2)) ||
 			((parts.size() > 2) && (m_formspec_version > FORMSPEC_API_VERSION))) {
-		parseColorString(parts[0], m_bgcolor, false);
-
-		if (parts.size() == 2) {
-			std::string fullscreen = parts[1];
-			m_bgfullscreen = is_yes(fullscreen);
+		if (parts.size() == 1) {
+			if (current_parent == this)
+				parseColorString(parts[0], m_bgcolor, false);
+			else {
```

Comment:
Brackets for upper condition required (by code style).

Reply:
Done.